pkg/porter/explain.go

```diff
@@ -230,6 +231,7 @@ func generatePrintable(bun cnab.ExtendedBundle, action string) (*PrintableBundle
 	sort.Sort(SortPrintableCredential(pb.Credentials))
 
 	for p, v := range bun.Parameters {
+		v := v // Go closures are funny like that
```

Comment:
@carolynvs May I know the reason behind this one? Might be something that I haven't known yet about Go 😕

Reply:
Yeah it's super obscure and a really annoying part of how Go handles "closures". If you search for "golang for loop closure" you'll find a lot of articles on it.

Without this hack, when I save off a reference to `v` on the display parameter, it ends up with all parameters having a reference to the same address instead of each display parameter getting a reference to its proper parameter.

Reply:
Ah, I just realized they mentioned it on the official repo. It's this [one](https://github.com/golang/go/wiki/CommonMistakes#using-reference-to-loop-iterator-variable), right? Thank you for the nugget of knowledge, Carolyn! Appreciate it 😄

Reply:
Yup! That is the problem I was working around. 👍